app-arch/zoo/zoo-2.10-r5.ebuild

```diff
@@ -1,31 +1,32 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit toolchain-funcs
 
 DESCRIPTION="Manipulate archives of files in compressed form"
 HOMEPAGE="https://packages.debian.org/sid/utils/zoo"
 SRC_URI="http://http.debian.net/debian/pool/main/z/${PN}/${PN}_${PV}.orig.tar.gz
 	http://http.debian.net/debian/pool/main/z/${PN}/${PN}_${PV}-28.debian.tar.xz"
+S="${WORKDIR}"/${P}.orig
 
 LICENSE="public-domain"
 SLOT="0"
 KEYWORDS="~alpha amd64 ~arm ~hppa ppc ppc64 sparc x86 ~amd64-linux ~x86-linux ~sparc-solaris ~x86-solaris"
-IUSE=""
 
-S="${WORKDIR}/${P}.orig"
+PATCHES=(
+	"${WORKDIR}"/debian/patches/.
```

Comment:
@SoapGentoo preference here? Expand or is this fine?

I’ve been expanding recently but not sure I like it or not. Makes it easier to skip one though.

Reply:
I'm fine with both. If we always wholesale apply all patches, I'd keep it like this, but again, no preference